docs/developer/tutorials/logs/submit-logs-from-http.md

```diff
@@ -0,0 +1,142 @@
+# Submit Logs from HTTP API
+
+## Getting Started
+
+This tutorial assumes you have done the following:
+
+- [set up your environment](../../index.md#getting-started)
+- read the [logs crawler documentation](../../base/logs-crawlers.md)
+- read about the [HTTP capabilities](../../base/http.md) of the base class
```

Comment:
```suggestion
- [Set up your environment](../../index.md#getting-started)
- Read the [Logs Crawler documentation](../../base/logs-crawlers.md)
- Read about the [HTTP capabilities](../../base/http.md) of the base class
```

Is the Logs Crawler docs fully flushed out or are these steps place holders right now?

Reply:
The Logs Crawler docs are "good enough"™️ at this point.
nit, but if we're starting the bullet points with uppercase, we should be ending them with periods ;)

Reply:
Turned the bullets into full sents.